model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -48,6 +49,8 @@
     'Subtract': Sub,
     'VariadicSplit': VariadicSplit,
     'Clamp': AttributedClamp,
+    'DFT': DFT,
```

Comment:
Please, don't add these operations in this list, instead, add their base class `FFTBase` to `update_registration(classes=[Op, Activation, Elementwise, UnaryElementwise, LogicalElementwise, EmbeddingBagBase, ReduceOp, Scatter, ScatterNDBase, FFTBase], enabled_transforms=[], disabled_transforms=[])` function call in line 65 in this file. Contact me if you have any questions about it.

Reply:
Fixed.